cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt

```diff
@@ -411,15 +411,19 @@ private class CampaignManagerImpl(
             val localStateSize = messagesOptimizedLocalState?.jsonObject?.size ?: 0
             val isV6LocalStatePresent = dataStorage.preference.all.containsKey(LOCAL_STATE)
             val isV6LocalStatePresent2 = dataStorage.preference.all.containsKey(DataStorage.LOCAL_STATE_OLD)
-            val res =
-                ((gdprUUID != null || ccpaUUID != null) && localStateSize == 0) || isV6LocalStatePresent || isV6LocalStatePresent2
+            val isRequireNewConsentData = dataStorage.localDataVersion < DataStorage.HARDCODED_LOCAL_DATA_VERSION
+            val res = ((gdprUUID != null || ccpaUUID != null) && localStateSize == 0) ||
```

Comment:
Can you rename the variable `res` to something more descriptive?

Reply:
@andresilveirah I think we can rename a couple of other params to make the whole check more descriptive and readable